packages/writing-style/vale-styles/commercetools/Brand.yml

```diff
@@ -15,3 +15,12 @@ swap:
   SPHERE: the commercetools platform
   SPHERE.IO: the commercetools platform
   sphere.io: the commercetools platform
+  MC: Merchant Center
+  merchant center: Merchant Center
+  Merchant center: Merchant Center
+  merchant Center: Merchant Center
+  Custom Applications: custom applications
+  Custom applications: custom applications
+  custom Applications: custom applications
+  custom apps: custom applications
+  Custom Apps: custom applications
```

Comment:
Let me know if these make sense to you. Not sure if "custom applications" should also be "Custom Applications" or better lowercase.

Reply:
Good Question, It's a branding policy decision.  Technically it's correct to lowercase it unless we decide to declare them a product with an own name. 

ask @coolmilo ?

Reply:
Plus, I suggest to also discourage "MC apps"  in this list.

Reply:
Ah thanks